core/commons.yaml

```diff
@@ -260,9 +266,13 @@ components:
             bbox:
               description: |-
                 One or more bounding boxes that describe the spatial extent of the dataset.
-                In the Core only a single bounding box is supported. Extensions may support
-                additional areas. If multiple areas are provided, the union of the bounding
-                boxes describes the spatial extent.
+                
+                The first bounding box always describes the overall spatial
+                extent of the data. All subsequent bounding boxes can be
+                used to provide a more precise description of the extent and
+                identify clusters of data.
+                Clients only interested in the overall spatial extent will
+                only need to access the first item in each array.
```

Comment:
```suggestion
                
                The first bounding box describes the overall spatial
                extent of the data. All additional bounding boxes describe 
                more precise extents, e.g., to identify clusters of data.
                Clients only interested in the overall spatial extent will
                only need to access the first item in each array.
```

Reply:
I will make that change in the OGC API Features pull request, too.